### PR TITLE
Update release-4.0.12.md

### PR DESCRIPTION
**Pull request by overvenus**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-4.0.12.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ TiDB version: 4.0.12
     + TiCDC
 
         - Add a double confirmation when users create or resume the changefeed with the `start-ts` or `checkpoint-ts` 1 day before the current timestamp [#1497](https://github.com/pingcap/ticdc/pull/1497)
+        - Add grafana panels for old value [#1571](https://github.com/pingcap/ticdc/pull/1571)
```
Comment (TomShawn) [inline releases/release-4.0.12.md:55]:
```suggestion
        - Add Grafana panels for the Old Value feature [#1571](https://github.com/pingcap/ticdc/pull/1571)
```

File: releases/release-4.0.12.md [hunk 2]
```diff
@@ -119,6 +120,9 @@ TiDB version: 4.0.12
 
         - Fix a disorder issue of the `resolved ts` event [#1464](https://github.com/pingcap/ticdc/pull/1464)
         - Fix a data loss issue caused by wrong table scheduling due to the network problem [#1508](https://github.com/pingcap/ticdc/pull/1508)
+        - Fix a bug of untimely release of resources when stopping a processor [#1547](https://github.com/pingcap/ticdc/pull/1547)
+        - Fix a bug that txn counter is not correctly updated which may cause database connection leak [#1524](https://github.com/pingcap/ticdc/pull/1524)
```
Comment (TomShawn) [inline releases/release-4.0.12.md:124]:
```suggestion
        - Fix a bug that the transaction counter is not correctly updated, which might cause database connection leak [#1524](https://github.com/pingcap/ticdc/pull/1524)
```

File: releases/release-4.0.12.md [hunk 3]
```diff
@@ -119,6 +120,9 @@ TiDB version: 4.0.12
...
+        - Fix multiple owner could co-exist when pd has jitter, which may lead to table missing [#1540](https://github.com/pingcap/ticdc/pull/1540)
```
Comment (TomShawn) [inline releases/release-4.0.12.md:125]:
```suggestion
        - Fix the issue that multiple owners can co-exist when PD has jitter, which might lead to table missing [#1540](https://github.com/pingcap/ticdc/pull/1540)
```

File: releases/release-4.0.12.md [hunk 4]
```diff
@@ -119,6 +120,9 @@ TiDB version: 4.0.12
 
         - Fix a disorder issue of the `resolved ts` event [#1464](https://github.com/pingcap/ticdc/pull/1464)
         - Fix a data loss issue caused by wrong table scheduling due to the network problem [#1508](https://github.com/pingcap/ticdc/pull/1508)
+        - Fix a bug of untimely release of resources when stopping a processor [#1547](https://github.com/pingcap/ticdc/pull/1547)
```
Comment (qiancai) [inline releases/release-4.0.12.md:123]:
```suggestion
        - Fix a bug of untimely release of resources after a processor is stopped [#1547](https://github.com/pingcap/ticdc/pull/1547)
```

**Comment by TomShawn** [comment]
Cherry-picked PRs:
+ https://github.com/pingcap/docs/pull/5196
+ https://github.com/pingcap/docs/pull/5195

**Pull request merged**